user_immutable/models/res_users.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, models
+from odoo.exceptions import AccessError
+
+IMMUTABLE = 'user_immutable.group_immutable'
+
+
+class ResUsers(models.Model):
+
+    _inherit = 'res.users'
+
+    def _check_immutable(self):
+        """ Check to see if the user being edited is Immutable and if so,
```

Comment:
I always screw these up. I even looked at that PEP earlier that day. (retardedsadpanda)